### ISSUE TITLE
OnNotificationSent Feature

### ISSUE THREAD

**Issue by Dorgam** [EddyVerbruggen/nativescript-local-notifications#76]
Currently, we can only specify app behavior after the user clicks on the notification. It would be more flexible if we can specify a certain behavior after a notification is sent, like scheduling another notification with a different period or changing something in the app itself.

Thanks.

**Comment by aop4** [comment]
I totally agree. (More generally, NativeScript would benefit from a way to schedule the execution of JavaScript code in a cross-platform way, even when the CPU is asleep. But this would be a great place for that feature!)

**Comment by Firetrip** [comment]
This would be really great. We struggle a lot at the moment to run some tiny code at a future timestamp (e.g. when a scheduled message is sent).
